### PR TITLE
Add description of postprocessed results

### LINKED ISSUE #54: Postprocessed data is not documented
The variable names that appear in postprocessed data should be documented. In the course of that, we can check if the naming system is consistent.

### PR DESCRIPTION
**Pull request by MaGering**
Resolves  #54

This PR provides a description of the postprocessed data including its directory structure and the files it contains.

### CODE REVIEW

File: docs/overview.rst [hunk 1]
```diff
@@ -45,5 +45,27 @@ set the values of parameters for all components.
...
+    postprocessed
+    ├── sequences
+    │     ├── bus
+    │     ├── by_variable
+    │     ├── component
+    ├── objective.csv
+    ├── scalars.csv
+
+The directory `sequences` contains time series of flows. In directory `bus` flows from and to each bus can be found.
+In the directory `by_variable` a `csv` file contains all time series of the flows and another `csv` file contains the
```
Comment (jnnr) [inline docs/overview.rst:61]: Most of the time, it is these two. Generally, it is any optimisation variable.

**Review by jnnr — COMMENTED** [review]
Looks good, thank you! Just one comment.

File: docs/overview.rst (hunk 1 above)
Comment (MaGering) [reply docs/overview.rst]: Thank you for this hint. How about the change I made in commit https://github.com/rl-institut/oemoflex/pull/76/commits/ba0e0f5b5ad2e0ec2c4ac4e4f4a56e5f7b0f1339?

Comment (jnnr) [reply docs/overview.rst]: Better! Thanks.

**Pull request merged**